packages/react-bootstrap-table2/package.json

```diff
@@ -38,7 +38,7 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "react-transition-group": "^4.2.0",
-    "underscore": "1.9.1"
+    "underscore": "1.12.1"
```

Comment:
Shouldn't we put "`^1.12.1`" instead so future security updates get automatically pushed in?

Reply:
This PR was auto created by dependabot so not sure if CR comments are too relevant. And I don't think it should block merging this when there is high security threat reported by `npm audit` which can cause CI failure on projects depending on this package. This PR is stalled for almost 2 months, can this please be merged?